src/main/scala/io/movies/services/MovieService.scala

```diff
@@ -0,0 +1,31 @@
+package io.movies.services
+
+import io.movies.algebras.Repository
+import io.movies.model.{Movie, RegisteredMovie}
+
+trait MovieService[F[_]]{
+  def getMovieById(id: Short): F[Option[RegisteredMovie]]
+  def getMovies: F[List[RegisteredMovie]]
+  def addMovie(movie: Movie): F[Int]
+}
+
+object MovieService{
+  def impl[F[_]](implicit repository: Repository[F]) =
+    new MovieService[F] {
+      override def getMovieById(id: Short): F[Option[RegisteredMovie]] = {
+        repository.getMovieById(id)
+      }
+
+      override def getMovies: F[List[RegisteredMovie]] = {
+        repository.getMovies
+      }
+
+      override def addMovie(movie: Movie): F[Int] = {
+        repository.addMovie(movie)
+      }
+    }
+}
+
+  // Con otra tabla de directores, se podría primero acceder a esa por nombre, coger id y usar
+  // ese id en la tabla movie
```

Comment:
Please write the comment in English

Reply:
See https://github.com/gutiory/MoviesServer/pull/1/commits/8f548691a4a0d10915d3b88a0a0f9f2f33e7cdb1